app/models/service.rb

```diff
@@ -156,7 +156,11 @@ def request_type
   end
 
   def retireable?
-    parent.present? ? true : type.present?
+    if parent.present? && provisioned?
+      true
+    else
+      type.present?
```

Comment:
I guess I don't understand this part of the conditional, but I think the part you changed looks good.

Reply:
https://github.com/ManageIQ/manageiq/pull/17317#discussion_r186528878

Reply:
I think this line deserves a comment since it is really difficult to understand the basis for it without more information.  Maybe a comment and the link you provided above.

Reply:
Even with a comment no one is going to like it, but fine.